src/script/page/LeftSidebar/panels/Preferences.test.tsx

```diff
@@ -44,19 +41,19 @@ describe('Preferences', () => {
     expect(queryByText('preferencesAV')).toBeNull();
   });
 
-  it('renders the about section in a web app', () => {
+  it('renders the a/v section in a desktop app', () => {
```

Comment:
The refactor in the `Preferences.tsx` is what fixes the tests, the shuffling around here is only because the test case description were not accurate